gatsby/src/components/DrugSections/InteractionsContent/InteractionsContent.test.tsx

```diff
@@ -4,13 +4,13 @@ import { InteractionsContent } from "./InteractionsContent";
 
 describe("InteractionsContent", () => {
 	describe("Single interactant", () => {
-		const interactants = [{ slug: "abacavir", title: "<em>Abacavir</em>" }];
+		const interactants = [{ slug: "abacavir", title: "Aba<em>cavir</em>" }];
```

Comment:
Interesting! Not saying it's wrong, just curious about why the emphasis on the second part of the word?

Reply:
it's because there aren't (in reality) html element at the start of a drug name, so it was a more real-world test, that and the `decapitalize` function doesn't take into account HTML

Reply:
Ta!